iree/vm/test/emitc/module_test.cc

```diff
@@ -18,9 +18,7 @@
 #include "iree/base/status.h"
 #include "iree/testing/gtest.h"
 #include "iree/vm/api.h"
-
-// This would be generated together with the functions in the header
-#include "iree/vm/test/emitc/arithmetic_ops.h"
+#include "iree/vm/test/emitc/arithmetic_ops.module"
```

Comment:
The `arithmetic_ops.module` actually is a header file but at the moment it is created via `iree_bytecode_module`. I will take care of this workaround in a separate PR so that we don't have to rely on `iree_bytecode_module`.

Reply:
👍 cool was about to ask about the lack of a .h, but this makes sense!